src/components/organisms/plan/plan.html

```diff
@@ -0,0 +1,54 @@
+<div class="orga-plan">
```

Comment:
```suggestion
<div class="organisms-plans">
```
極めて一般的な略称でない限りあまり単語の省略はしないほうがいいと思います。